### DISCUSSION TITLE
DNS config on Vercel side for webmail

### DISCUSSION THREAD

**Discussion by Serverket** [vercel/community#3831] [Feature Question] [answered]
### Question

I have my provider's Nameservers configured to point to Vercel because I want Vercel to deploy the code from my website (from the repo) instead of the hosting service from my provider, that's why I didn't acquire a plan for hosting. However, I used to configure CPanel to make DNS point to Vercel but at the same time make email to work with webmail DNS pointing to the provider, is possible to setup this on Vercel side? I need the DNS from the website to be managed with vercel, but yet I need the emails to work, what would be the solution? Thanks in advance.

**Comment by amyegan** [comment, marked as answer]
Hey @Serverket. You can [manage DNS Records](https://vercel.com/docs/projects/domains/managing-dns-records#) from the [Domains](https://vercel.com/dashboard/domains) page. You may need to check with your webmail provider for the exact records needed, but it sounds like you already know them. From there, you can use one of the available [email DNS presets](https://vercel.com/guides/using-email-with-your-vercel-domain#dns-preset) or manually [add DNS Records](https://vercel.com/guides/using-email-with-your-vercel-domain#adding-dns-records).

**Reply by Serverket** [reply]
Hi @amyegan, thank you so much for taking the time to give me such clear and helpful reply! It's done, I really appreciate your help, I managed to change all the DNS config through the Vercel dashboard as you suggested, [this guide](https://www.namecheap.com/support/knowledgebase/article.aspx/1340/2176/namecheap-private-email-records-for-domains-with-thirdparty-dns/) gave me the records I needed for Private Email to work since it is what my provider uses instead of webmail, and it works!
